src/main/java/org/jitsi/videobridge/BitrateController.java

```diff
@@ -251,6 +303,21 @@ public void update(List<Endpoint> conferenceEndpoints)
             }
         }
 
+        long availablePaddingBps = bweBps - mediaBps;
+        paddingParams.bps = Math.min(optimalBps - mediaBps, availablePaddingBps);
+        assert paddingParams.bps >= 0;
```

Comment:
Isn't the "paddingParams.bps < 0" a valid case which could happen if "mediaBps > bweBps"?